bindings/gumjs/gumquickmemory.c

```diff
@@ -376,6 +378,20 @@ GUMJS_DEFINE_FUNCTION (gumjs_memory_protect)
   return JS_NewBool (ctx, success);
 }
 
+GUMJS_DEFINE_FUNCTION (gumjs_memory_query_protection)
+{
+  gpointer address;
+  GumPageProtection prot;
+
+  if (!_gum_quick_args_parse (args, "p", &address))
+    return JS_EXCEPTION;
+
+  if (!gum_memory_query_protection (address, &prot))
+    return JS_EXCEPTION;
```

Comment:
Need to throw a JS exception here. (Would add a label with that logic, like we do elsewhere.)